src/server_manager/messages/master_messages.json

```diff
@@ -1243,9 +1243,69 @@
       }
     }
   },
-  "share_invite": {
-    "message": "You\u2019re invited to connect to my Outline server. Use it to access the open internet, no matter where you are. Follow the instructions on your invitation link below to download the Outline App and get connected.",
-    "description": "This string appears as a paragraph within a dialog that provides an invitation to share access to a server. Belongs to the instructions for the user with whom access is being shared."
+  "share_invite_intro": {
+    "message": "You’re invited to Outline! Follow these instructions to use my server to access the open internet on any device, no matter where you are. No personal information will be collected.",
+    "description": "TODO"
```

Comment:
Still need to fill these in. Requesting a review in advance due to time constraints.